File: lab2_PPOIS/MailSystem_post/Human/Human.cpp
#include "Human.h"

Human::Human(string name, string surname, Date birthday, string passportID)
	:name(name), surname(surname), birthday(birthday), passportID(passportID) {};


bool Human::VerifyIdentity(const string& passID) const {
	return passportID == passID;
}

string Human::GetFullName() {
	return surname + " " + name;
}

File: PPOIS/sem4/lab1/models/human.py
from .date import Date

class Human:
    def __init__(self,name:str,surname:str,bday:Date):
        self.__name=name
        self.__surname=surname
        self.__bday=bday


    def __str__(self):
        return f"Full name: {self.full_name} \nBirthday: {self.__bday}"


    @property
    def full_name(self):
        return self.__surname + " " + self.__name